x-pack/elastic-agent/CHANGELOG.asciidoc

```diff
@@ -21,6 +21,8 @@
 - Fix issues when running `mage package` for all the platforms. {pull}17767[17767]
 - Rename the User-Agent string from Beats Agent to Elastic Agent. {pull}17765[17765]
 - Remove the kbn-version on each request to the Kibana API. {pull}17764[17764]
+- Fixed process spawning on Windows {pull}17751[17751]
```

Comment:
This is a merge conflict with the changelog?

Reply:
yeah i dont unerstand this. it;s not in 7.x it's from different PR, i need to cleanup

Reply:
@michalpristas yeah, it's common I make the mistake all the times with CHANGELOG 9 out of 10 I have to make some change to it. I try always to look that file on the cherry-pick PR because even if you don't have a conflict it may content more items..